lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,47 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server:                 # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
```

Comment:
The more important part is that those namespaces won't work if you want to change *part* of them in an environment. For example:
```ruby
>> config = YAML.load("
default: &default
  paths:
    source: app/javascript
    entry: packs
    output: packs

production:
  <<: *default

  paths:
    output: foo
")
=> {"default"=>{"paths"=>{"source"=>"app/javascript", "entry"=>"packs", "output"=>"packs"}}, "production"=>{"paths"=>{"output"=>"foo"}}}
>> config["production"]["paths"]["source"]
=> nil
>> config["production"]["paths"]["entry"]
=> nil
>> config["production"]["paths"]["output"]
=> "foo"
```